packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -27,6 +27,11 @@ const propTypes = {
    * Ref callback for the scrollable area of the content container.
    */
   scrollRefCallback: PropTypes.func,
+  /**
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)
+   * Sets focus on content when set to `true`. Focus on content helps in scrolling  within container when there is no interactive element to focus within container.
+   */
+  setFocusOnContent: PropTypes.bool,
```

Comment:
can we rename this ? this is usefull when there is a scroll over content but there is no interactive element. it can be something like setfocusoncontentcontainer or setfocusoncontainer.